bakerydemo/base/blocks.py

```diff
@@ -60,6 +61,22 @@ class Meta:
         template = "blocks/blockquote.html"
 
 
+class DetailsBlock(StructBlock):
+    """
+    Custom `StructBlock` for creating a details block with summary, content, and open fields.
+    """
+
+    summary = CharBlock(required=True)
+    content = RichTextBlock(required=True)
+    open = BooleanBlock(
+        required=False, default=True, label="Open", help_text="Open by default"
```

Comment:
I’m not too clear why we are using a `<details>` element in HTML if it’s to have those blocks opened by default. Could we default to "False" instead?

I’d also suggest removing the label if it just repeats the field identifier, and writing a more descriptive help text or removing it (stating the default value isn’t too helpful).